File: Poin B.java/Processor.java
public class Processor {
    //atribut
    private String name;
    private int price;

    //konstruktor
    public Processor(String name, int price) {
        this.name = name;
        this.price = price;
    }
    public Processor() {
    }

    //get set methods
    public int getPrice() {
        return price;
    }
    public String getName() {
        return name;
    }
    public void setName(String name) {
        this.name = name;
    }
    public void setPrice(int price) {
        this.price = price;
    }

    //output
    public void output(){
        System.out.println("Processor               : " + this.getName());
        System.out.println("Price                   : " + this.getPrice());
    }

    
    
}

File: Poin B.py/Processor.py
class Processor:
    #atribut
    name = ""
    price = 0
    
    #constructor
    def __init__(self, n = "", p = 0) :
        self.name = n
        self.price = p
    
    #get set methods
    def setName(self, n= "") :
        self.name = n
        
    def setPrice(self, p = 0) :
        self.price = p
        
    def getName(self) :
        return self.name
    
    def getPrice(self) :
        return self.price
    
    def output(self) :
        print("Processor                    : " + str(self.getName()))
        print("Price                        : " + str(self.getPrice()))
        
    
    
        
    